### DISCUSSION TITLE
How do you make the opponent scare BF and GF?

### DISCUSSION THREAD

**Discussion by BrandonHatesAnime** [KadeArchive/Kade-Engine#2440] [Q&A]
Just like in Mid Fight Masses, Ruv scared GF while he was singing.
The lightning strike in Week 2 scared BF and GF.
How do I make it so that whenever the opponent sings it scares the BF and GF?

**Comment by bwkam** [comment]
Using modcharts with lua I guess

**Comment by Jotaro-Gaming** [comment]
cursteps modcharts character switch alt animation and finally make them scared the whole time but if u want to animate them with them being scared just go to the boyfriend fla
